### DISCUSSION TITLE
Groups in 3dec file

### DISCUSSION THREAD

**Discussion by mqc007** [neperfepx/neper#475] [Q&A] [answered]
HI，

I asked yesterday and learned how to group the first scale grain, and i achieved the visualize the group in neper. 

Here is my code 

`neper -T -n 5::5 -domain "cube(5,5,5)" -reg 1 -group "(scaleid(1))" -format tess,3dec -o 051202`

`neper -V 051202.tess -datacellcol group -print 051202`

it works well in neper and im very happy to see the image colored in groups,  BUT when i open the .3dec format file in 3dec these group code didnt work. So i checked the command code in vscode and there is no command about group. 

although i know it's the problem about 3dec, im wondering if i used the group code correctly and i curious about the .3dec format command not having the group command. is there any possible the make .3dec format contain the group command? it's very important for me. 

thank you in advance. 
all regards.

![image](https://user-images.githubusercontent.com/41913110/168089055-fd7933ec-4dac-46a7-a77e-30bb22a3a452.png)

**Comment by rquey** [comment, marked as answer]
Neper does not write information about groups to the 3dec file.

What would be the syntax to include group information?
